app/models/document.rb

```diff
@@ -341,4 +342,16 @@ def set_defaults
     self.document_status = "discovered" unless document_status
     self.accessibility_recommendation = DEFAULT_DECISION unless accessibility_recommendation
   end
+
+  def set_complexity
+    if (number_of_tables.nil? || number_of_images.nil?) && document_category != "Form"
+      return
+    end
+    self.complexity = calculate_complexity
+  end
+
+  def calculate_complexity
```

Comment:
Just of a question: what's the benefit of having two separate functions `set_complexity` & `calculate_complexity` vs one that does both? Not a blocker.